r-package/grf/tests/testthat/test_validation.R

```diff
@@ -0,0 +1,67 @@
+library(grf)
+library(testthat)
+set.seed(1234)
+
+test_that("example data passes validation"){
```

Comment:
I don't think we need this test method, as the behavior it tests (good data passing validation) is extensively covered by other tests.

Reply:
removed.